yb-voyager/cmd/importData.go

```diff
@@ -903,6 +905,35 @@ func doOneImport(task *SplitFileImportTask, targetChan chan *tgtdb.Target) {
 	}
 }
 
+func addCustomSessionVars(conn *pgx.Conn) bool {
+	sessionVarsPath := exportDir + "/metainfo/yb_session_vars.sql"
+	if _, err := os.Stat(sessionVarsPath); errors.Is(err, os.ErrNotExist) {
+		return false
+	}
+	utils.PrintAndLog("Using custom session variables from %s.", sessionVarsPath)
+	varsFile, err := os.Open(sessionVarsPath)
+	if err != nil {
+		utils.ErrExit("Error while opening yb_session_vars.sql: %v", err)
+	}
+	defer varsFile.Close()
+	fileScanner := bufio.NewScanner(varsFile)
+	fileScanner.Split(bufio.ScanLines)
```

Comment:
what does this line means?

Reply:
Split the file using the delimiter as \n

Reply:
Afaik this is not required, `fileScanner.Text()` function does split on newline char by itself.